src/tests/ftest/slurm_setup.py

```diff
@@ -162,6 +158,10 @@ def start_munge(args):
         args (Namespace): Commandline arguments
 
     """
+    if not args.sudo:
+        sudo = ""
+    else:
+        sudo = "sudo"
```

Comment:
```suggestion
    if args.sudo:
        sudo = "sudo"
    else:
        sudo = ""
```
This is easier to understand IMO.